### DISCUSSION TITLE
Swap target and source branch order when browsing PRs

### DISCUSSION THREAD

**Discussion by mtzaquia** [community/community#137139] [Mobile]
When quickly scanning PRs, I often miss having the target branch visible at a glance.

In our setup, that made me miss that a PR was aiming `candidate`, rather than `develop` a couple of times.

So my suggestion is to swap the order: target branch followed by source branch (also, it's arguably less relevant to know which name an engineer decided to pick for their working branch).

![image](https://github.com/user-attachments/assets/8ca9cb0a-6259-437b-a0ce-e67e667d3cf7)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.175.0 (139716900)
Device: iPhone 14 Pro (iOS 18.0)
</details>

**Comment by javiercarpenterr** [comment]
This is a great suggestion. I've definitely missed that a PR was targeting the 'master' branch because the source branch was listed first. Swapping the order would definitely be an improvement.
